docs/source/conf.py

```diff
@@ -48,24 +57,25 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'Jupyter NBConvert'
-copyright = 'Jupyter Development Team'
+project = 'nbconvert'
+copyright = '2015, Jupyter Development Team'
+author = 'Jupyter Development Team'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-_version_py = '../../nbconvert/_version.py'
-version_ns = {}
-exec(compile(open(_version_py).read(), _version_py, 'exec'), version_ns)
 # The short X.Y version.
-version = '%i.%i' % version_ns['version_info'][:2]
+version = '4.0'
```

Comment:
Hmm, probably this should use the code that was there before so the version gets automatically updated.